### DISCUSSION TITLE
[Feature] Wrapping grid for Jobs page

### DISCUSSION THREAD

**Discussion by dzg** [immich-app/immich#11620] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Not sure where to put the code for a PR, but I would tweak the CSS for the Job Status page with this
```
#setting-content .gap-7 {
    display: grid;
    grid-template-columns: repeat(auto-fit, minmax(500px, 1fr));
    gap: 1rem;
}

#setting-content > section {
    width: 100%;
}
```
so that we can see all jobs on one page like:
<img width="1873" alt="image" src="https://github.com/user-attachments/assets/13c053b8-343a-4c31-9df0-d5095132f14d">

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5402
